daffodil-cli/src/test/scala/org/apache/daffodil/exi/TestEXIEncodeDecode.scala

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-package org.apache.daffodil.saving
+package org.apache.daffodil.exi
```

Comment:
I guess my IDE has decided that directory names and packages must conform. 
I think the right thing here is for the dirctory name to be "saving", and the package to match.

Reply:
I think the `saving` package is intended for the `daffodil save-parser` command. I think the exi package name is probably fine. 

Alternatively, we could just get rid of all these separate packages. I don't think we really gain anything by having each test suite it a unique package. We could just move all of our scala files in a "cli" directory and then all have the package names be `org.apache.daffodil.cli` or something more generic.

Reply:
I second Steve's suggestion to remove the extra packages below org.apache.daffodil.cli.